planning.md

```diff
@@ -34,57 +34,92 @@ title:  "Processing Manual - Planning"
 
 ### Step 6 - Sharing the Project Vitals
 
-- Once the Project Vitals document is submitted, discussed, and approved, it is shared by the Assistant Director for Processing with the Archives Management Team and the Processing Team.
+- Once the Project Vitals document is submitted, discussed, and approved, the Assistant Director for Processing will share it with the Archives Strategy Group and the Processing Team.
 
-#### Project Vitals Template
+#### Project Vitals Template for Processing Projects
 
-1. Physical Vitals
+1. Supporting Values
+   - How does this work support RAC institutional values?
+   - Please highlight one or more values and how you will consider them while conducting this project.
+2. Physical Vitals
     - Volume (cubic feet) and/or estimated volume of digital files (including     
         estimated number of digital files and MB/GB)
     - Current number of boxes
     - Current Location
     - Estimate a range of processed boxes (when reboxing to document boxes)
         - Estimate 2.5 - 2.75 letter-size document boxes for each record carton.
-2.  Accessioning Vitals
+3.  Accessioning Vitals
     - Accession Number
     - Accession Level
     - Have any applicable born-digital files been ingested or accessioned?
-3.  Descriptive Vitals
+4.  Descriptive Vitals
     - Collection/Finding Aid Title
     - Is there an initial inventory?
     - If so, is the inventory available in ArchivesSpace?
     - List any recommendations for the creation of RAC-optional notes.
-4.  Preservation Vitals
+5.  Preservation Vitals
     - Is existing housing acceptable to retain?
     - Document any significant rehousing or preservation issues.
     - Are any specialized supplies needed?
-5.  Processing Vitals
+6.  Processing Vitals
     - Conditions of Access
     - Establishing Archival Context and Structure
         - What are these records/papers?
         - Who created them?
         - What do they mean or represent?
         - How, or by what process, were they created or maintained? (Together? Separate?)
         - Are there any biases, institutional or personal, that might apply to the material?
-        - Arrangement
-            - Standard – Original order/As received
+    - Arrangement
+        - Standard – Original order/As received
             - If NOT Standard – Propose an Arrangement
     - Processing Level
-6.  Projected Work Time
+7.  Projected Work Time
+
+#### Project Vitals Template for substantial Redescription Projects
+
+1. Supporting Values
+   - How does this work support RAC institutional values?
```

Comment:
Could we link our values here? https://rockarch.org/about-us/mission-vision-values/#our-values